ci-operator/step-registry/rosa/cluster/upgrade/cluster/rosa-cluster-upgrade-cluster-commands.sh

```
@@ -320,11 +321,20 @@ if [[ "$HOSTED_CP" == "true" ]]; then
   HCP_SWITCH="--control-plane"
 fi
 
+current_version=$(rosa describe cluster -c $cluster_id -o json | jq -r '.openshift_version')
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)